### ISSUE TITLE
⚠️5.0.0 to 5.1.6 Deprecated - These versions may be blocked or rate limited. Please update to 5.1.8 or newer⚠️

### ISSUE THREAD

**Issue by jkasten2** [OneSignal/OneSignal-Android-SDK#2048]
## Notice
As we continue to improve our systems with the new User Model updates, we found a bug in our Android SDK 5.0.0 to 5.1.6 versions that is causing exceptional load on our API. Some API requests might be blocked during peak load on these versions of SDKs used in Android apps. This means new users may not be created and current user data may not be updated (like tags, last session, session count, etc) during peak times. The full list of impacted SDKs versions are:
* OneSignal-Android-SDK 5.0.0 to 5.1.6
* OneSignal-Unity-SDK 5.0.0 to 5.1.1
* OneSignal-Flutter-SDK 5.0.0 to 5.1.3
* OneSignal-DotNet-SDK 5.0.0 to 5.1.1
* OneSignal-Cordova-SDK 5.0.0 to 5.1.0
* react-native-onesignal 5.0.0 to 5.1.0

To ensure your Android app isn't impacted, we ask that you upgrade to the latest version of our SDKs and release the updated version of your app as soon as possible. Please make sure you update to at least one of the following versions or newer:
* [OneSignal-Android-SDK 5.1.8](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/5.1.8)
* [OneSignal-Unity-SDK 5.1.2](https://github.com/OneSignal/OneSignal-Unity-SDK/releases/tag/5.1.2)
* [OneSignal-Flutter-SDK 5.1.4](https://github.com/OneSignal/OneSignal-flutter-SDK/releases/tag/5.1.4)
* [OneSignal-DotNet-SDK 5.1.2](https://github.com/OneSignal/OneSignal-DotNet-SDK/releases/tag/5.1.2)
* [OneSignal-Cordova-SDK 5.1.1](https://github.com/OneSignal/OneSignal-Cordova-SDK/releases/tag/5.1.1)
* [react-native-onesignal 5.1.1](https://github.com/OneSignal/react-native-onesignal/releases/tag/5.1.1)

_Android apps on SDKs older than 5.0.0 are not affected._

**Comment by berkaytikenoglu** [comment]
Did you find the solution?

**Comment by emawby** [comment]
@berkaytikenoglu Yes the fix was released in 5.1.7 and more performance enhancements have been shipped since then
